src/utilities/fetchCategories.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+const fetchCategories = () => axios.get('/api/categories');
```

Comment:
this needs error handling ! 
Rn the server is in your control but later down the road you have to be ready if the server doesnt work , please use try catch / axios error handling to make code robust.

Reply:
So I'm handling the error where I'm calling this function, using async await there so I could show that error using react toastify, will add it